lib/backburner.js

```diff
@@ -73,6 +78,44 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var args = Array.prototype.slice.call(arguments, 1);
```

Comment:
@asakusuma You need to pull the arguments off like so:

```
var args = new Array(argument.length - 1);
for (var i = 1, l = args.length; i < l; i++) {
  args[i - 1] = arguments[i]
}
```

Otherwise you leak the arguments object causing a deopt in V8.